jvm/src/main/scala/fm/common/BaseEncoding.scala

```diff
@@ -215,10 +215,15 @@ object BaseEncoding {
       this
     }
   }
-  
-  private val exceptionHandler: PartialFunction[Throwable,Nothing] = { 
```

Comment:
@tpunder I forget if there is a next-step on this PR.  This `GuavaBaseEncoding.DecodingException` will never get triggered based upon the current code.  So this either fixes the current logic, or we can stick with the current `IllegalArgumentException` and update this to remove the `exceptionHandler` to avoid confusion.